kuma/conftest.py

```diff
@@ -1,8 +1,85 @@
+from datetime import datetime
+from collections import namedtuple
+
 import pytest
 from django.conf import settings
 from django.core.cache import caches
 
+from kuma.wiki.models import Document, Revision
+
+
+BannedUser = namedtuple('BannedUser', 'user ban')
+
 
 @pytest.fixture()
 def cleared_cacheback_cache():
     caches[settings.CACHEBACK_CACHE_ALIAS].clear()
+
+
+@pytest.fixture
+def wiki_user(db, django_user_model):
+    """A test user."""
+    return django_user_model.objects.create(
+        username='wiki_user',
+        email='wiki_user@example.com',
+        date_joined=datetime(2017, 4, 14, 12, 0))
+
+
+@pytest.fixture
+def wiki_user_2(db, django_user_model):
```

Comment:
I'm 👎 on adding all the potentially useful fixtures. I see narrowly scoped fixtures as a feature of pytest, versus the Django fixtures which tended to grow to cover many, many use cases. I'd just pull up the ``wiki_user`` and ``root_doc`` fixtures, and leave the rest until they are needed by multiple modules.  It's a weak preference, and I could be persuaded.